assets/css/_media-queries.scss

```diff
@@ -8,7 +8,7 @@
       box-shadow: 0 0 0.5rem $shadow;
       flex: 3 3 0;
       height: auto;
-      min-width: 180px;
+      min-width: 300px;
       position: relative;
```

Comment:
Does the width here prevent the horizontal scroll? If so should we just hide the overflow-x?